src/Components/Web.JS/src/Platform/Circuits/DefaultReconnectDisplay.ts

```diff
@@ -29,11 +31,13 @@ export class DefaultReconnectDisplay implements ReconnectDisplay {
     ];
 
     this.modal.style.cssText = modalStyles.join(';');
-    this.modal.innerHTML = '<h5 style="margin-top: 20px"></h5><button style="margin:5px auto 5px">Retry?</button>';
+    this.modal.innerHTML = '<h5 style="margin-top: 20px"></h5><button style="margin:5px auto 5px">Retry?</button><p>Alternatively, <a href>reload</a></p>';
```

Comment:
Could we avoid the inline styles here?

I know @blowdart would prefer if we don't do this as it complicates things for people writing a CSP policy (forces them to add unsafe-inline or something like that which is bad)

Reply:
What he said :) Avoid inline styles and prefer CSS

Reply:
We can make it get set by JS instead of a style attribute, but don’t want to put this is .css for reasons.

This is not changing in this PR so can be addressed separately.